Source/QuartzCore additions/CGPathAdditions.m

```diff
@@ -21,7 +21,7 @@ CGFloat fixInfinity(CGFloat inputFloat){
     return inputFloat;
 }
 
-CGPoint *fixPointsInfinity(CGPathElement *element){
+CGPoint *fixPointsInfinity(const CGPathElement *element){
```

Comment:
Code cannot work with const, though.

I think ... "CGPoint *fixPointsInfinity(CGPathElement *element)" needs to be rewritten to:

1. allocate a new array
2. put changed values in new array, leaving old array as const
3. renamed to make it clear it allocates a new array

And "applyPathTranslation" has to be changed so that:

4. uses the new-named function
5. releases the new array at end of function